main.go

```diff
@@ -35,6 +35,9 @@ func main() {
 	vendorID := flag.String("vendor-id", "0x0", "VID of the camera to search (in hex), default=0x0 (all)")
 	productID := flag.String("product-id", "0x0", "PID of the camera to search (in hex), default=0x0 (all)")
 	maxResolution := flag.Bool("max-resolution", false, "change the resolution to the max (experimental)")
+	afInterval := flag.Int64("af-interval", 0, "disable or change the autofocus interval in seconds, default=0 (disabled)")
+	afFocusNow := flag.Int("af-focus-now", 0, "autofocus once on startup after X seconds, default=0 (disabled)")
```

Comment:
Most camera users will understand what "AF" stands for, but I'm slightly pessimistic about it. Let's make them more human-friendly. `autofocus-on-startup` is very long, but follows the description's sentence.

```suggestion
	afInterval := flag.Int64("autofocus-interval", 0, "disable or change the autofocus interval in seconds, default=0 (disabled)")
	afFocusNow := flag.Int("autofocus-on-startup", 0, "autofocus once on startup after X seconds, default=0 (disabled)")
```

Reply:
done